### ISSUE TITLE
Start of a loop misses notes

### ISSUE THREAD

**Issue by magnetophon** [sjaehn/BSchaffl#13]
As you can hear in BSchafflSeries.ardour from #12, every time ardour loops, there are a few notes missing in the first part of each repetition.
When you play the loop for the first time, the notes are not missing.

**Comment by sjaehn** [comment]
Sounds like something I observed before with Ardour loops and likely related to the problem we discussed with Robin in https://github.com/sjaehn/BSchaffl/issues/5. I'll check the console output. Then I can tell you more about it.

**Comment by sjaehn** [comment]
Not easy to understand again. Too many notes multiplied by four instances of B.Shaffl. You like complicated settings ;-).

I changed your BSchafflSeries.ardour a bit by assigning all MIDI notes to an individual pitch. This makes it possible to see what's wrong. My observations:

Within the loop.
Note 1 is played, then three notes are missing. The missing time is about the same as the plugins latency. If you lower the latency, you also reduce the number of missing notes. In the console output I can see that note 1 NOTE_ON is sent from the host (ardour) to the plugin before the change in time/position (due to the jump) is communicated to the plugin. Then the gap. This looks like the problem is related to problem we discussed before with Robin in #5.
